packages/DatePicker/src/DatePicker.js

```diff
@@ -44,14 +44,18 @@ const propTypes = {
   /** Callback when date is selected or input. */
   onChange: PropTypes.func.isRequired,
 
-  /** Errors callback */
+  /** Internal errors callback */
```

Comment:
Trying to be more explicit here using "Internal" to mean errors coming from inside the component